### ISSUE TITLE
Random Canon titles 71-73

### ISSUE THREAD

**Issue by riddella** [random-canon/random-canon#105]
Here are three Random Canon titles to enter. Be sure to use a first edition.

```
71. (#4026) Prince of Orange: An Historical Romance of the Dutch Protestant Revolution (1845), Google Books, 3 vol.
15 + 15 + 17 = 47 chapters; second title selected since first was a duplicate title by woman author
4026__chp46of47_seed4026.html

72. (#2664) Reade, A Simpleton (1873), 3 volumes, Gutenberg, Hathitrust
1 to 7, 8 to 17, 18 to 29 (continuous numbering); first title selected
2664__chp28of29_seed2664.html

73. (#7437) Yates, Two by Tricks (1874), 2 volumes, British Library
10 + 10 = 20 chapters; second title selected since first title already selected
7437__chp1of20_seed7437.html
```

**Comment by hanmills** [comment]
[4026__bentley_prince_of_orange_chp46of47_seed4026.txt](https://github.com/random-canon/random-canon/files/3221411/4026__bentley_prince_of_orange_chp46of47_seed4026.txt)
[2664__reade_a_simpleton_chp28of29_seed2664.txt](https://github.com/random-canon/random-canon/files/3221412/2664__reade_a_simpleton_chp28of29_seed2664.txt)
[7437__yates_two_by_tricks_chp1of20_seed7437.txt](https://github.com/random-canon/random-canon/files/3221413/7437__yates_two_by_tricks_chp1of20_seed7437.txt)

**Comment by ayarnell** [comment]
@riddella Text 7437 has a couple Ancient Greek words in it. What is the best way to enter these? I entered the Unicode characters, but of course the script does not like that.

**Comment by ayarnell** [comment]
@riddella 4026 and 2664 are in PR #123

**Comment by riddella** [comment]
@riddella Is the script giving you an error or just some warnings? If there are just warnings, I'd try to ignore them, given that you know the greek characters are correctly entered.

**Comment by riddella** [comment]
Merged 4026 and 2664 in #123 . 7437 remains.

**Comment by ayarnell** [comment]
@riddella Yes, it was just a warning. This text is in PR #130.

**Comment by riddella** [comment]
Closed by #130
